### PR TITLE
Refine table and tabs

### PR DESCRIPTION
**Pull request by janfaracik**
Really small/subtle pull request to refine the table and tabs components. These changes are more obvious in dark mode.

**Differences**
* Tightens up the padding, bringing it more inline with the recently introduced cards
* Slightly more rounded, again bringing it more inline with cards
* Borders are now visible when in dark mode (same as cards)

**Before**

image

**After**

image

**Before**

image

**After**

image

### Testing done

* Tables and tabs look as expected

### Proposed changelog entries

- Refine tables and tabs with reduced padding, rounded corners to match cards, and borders that are visible in dark mode.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [x] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [x] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [x] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [x] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@jenkinsci/sig-ux 

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [x] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [x] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [x] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [x] Proper changelog labels are set so that the changelog can be generated automatically.
- [x] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [x] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Comment by timja** [comment]
Could you add screenshots with multiple views (as I think that doesn't look the best atm)?

**Comment by janfaracik** [comment]
> Could you add screenshots with multiple views (as I think that doesn't look the best atm)?

Agree there - this MR hasn't changed how the tabs are displayed, more the background of them.

**Before**

image

**After**

image

**Before**

image

**After**

image

**Review by timja — APPROVED** [review]
LGTM

**Review by uhafner — APPROVED** [review]
Looks good (in the screenshots, I did not test it). 

Did you check if this has no negative impact on plugins that use Boostrap tabs or Jenkins tables in table column tooltips? The warnings and coverage plugins are 2 examples.

**Comment by mawinter69** [comment]
The screenshots look as if they were taken on top of #9410 because the sort arrows look different to what we have now

**Comment by janfaracik** [comment]
> The screenshots look as if they were taken on top of #9410 because the sort arrows look different to what we have now

Good spot - I must have been half asleep doing this. I've updated the screenshots, only difference is the arrows are correct now. Thanks!

**Comment by janfaracik** [comment]
> Looks good (in the screenshots, I did not test it).
> 
> Did you check if this has no negative impact on plugins that use Boostrap tabs or Jenkins tables in table column tooltips? The warnings and coverage plugins are 2 examples.

I'll take a look - thanks!

**Review by NotMyFault — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

File: war/src/main/scss/abstracts/_theme.scss
Comment (daniel-beck) [inline war/src/main/scss/abstracts/_theme.scss:178]: What's the unit change here for? <s>Weirdos</s> People who "zoom text only" on web pages?

Comment (timja) [reply war/src/main/scss/abstracts/_theme.scss:178]: This explains it well: https://stackoverflow.com/a/43131958

Generally in new web development `rem` is used pretty much exclusively (mostly) as it adapts to the users font-size

Comment (daniel-beck) [reply war/src/main/scss/abstracts/_theme.scss:178]: Could you be more explicit here? Because there's no difference for full-page zoom; only font-only zoom behaves different with this change.

**Pull request merged**
